broadcast/src/linked/signer/metrics.rs

```diff
@@ -55,57 +63,53 @@ impl Metrics {
             threshold: Counter::default(),
             new_broadcast: status::Counter::default(),
             rebroadcast: status::Counter::default(),
-            verify_duration: Histogram::new(
-                commonware_runtime::telemetry::histogram::Buckets::LOCAL.into_iter(),
-            ),
-            e2e_duration: Histogram::new(
-                commonware_runtime::telemetry::histogram::Buckets::NETWORK.into_iter(),
-            ),
+            verify_duration: histogram::Timed::new(verify_duration.clone(), Arc::clone(&clock)),
```

Comment:
I'm used to just seeing `clock.clone()` (which just clones the Arc) rather than this syntax

Reply:
Is there a particular reason you did it this way?

Reply:
Both ways are valid and do the same thing, I was just meaning to be clearer that the `Arc` was being cloned and not the `clock` itself. Maybe calling it `clock_arc` and then doing `clock_arc.clone()` would be a reasonable compromise. I don't feel strongly here, happy to call it `clock` and do `clock.clone()` as well